{"frames":[{"func":"print(imagem)","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":"`print.magick-image`(imagem)","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":"previewer(img)","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":"magick:::image_preview(x)","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":"image_write(img, path = tmp, format = format)","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":"writeBin(buf, path)","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":"file(con, \"wb\")","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0}],"message":"Error in file(con, \"wb\") : cannot open the connection\n"}